### PR TITLE
updated npm package

### LINKED ISSUE #1372: Cannot import model-viewer since 1.0.0 with npm install
### Description

As has been reported in a few other issues (https://github.com/google/model-viewer/issues/1300, https://github.com/google/model-viewer/issues/1208, maybe others), there's been a failure to resolve the model-viewer when using `import "@google/model-viewer"`. This worked fine until 1.0.0. In my case it is a vue based project, but I suspect it has the same root cause as the React based projects in other issues. It has something to do with the underlying bundler (Webpack, in my case).

As someone suggested in other threads, you can workaround this by using a direct import to the "dist" folder: `import "@google/model-viewer/dist/model-viewer"`. While this works, it is ugly and ultimately causes another issue.

In looking at the changes in 1.0.0, I traced it down to [this change to package.json](https://github.com/google/model-viewer/commit/92111b3ac68c22508bc00edb6300404407308008#diff-f4d9494fdb27b7fe22b92831ca57b757).

First, in the "files" section, "lib", and "src" are removed and the "main" and "module" now point to a non-existent location:

```
  "main": "./lib/model-viewer.js",
  "module": "./lib/model-viewer.js",
```
I manually edited this in my node_modules to use the "./dist" and could then import successfully with expected syntax.

Obviously, the primary problem is the inability to import, but a secondary problem I have is how we generate our 3rd party dependencies from Webpack seems to depend on this, in that I no longer get the transitive dependencies (i.e. three). I do not know if lib/src are required or not, but would like them to be restored, and would probably be needed to debug correctly.

#### Live Demo
Not available.

### Browser Affected
n/a

### OS
All - n/a

### Versions

- model-viewer: v1.0.0

### PR DESCRIPTION
**Pull request by elalish**
Fixes #1372 

In addition to cleaning up our dependencies and fixing our "main" pointer, this also ensures the .map files are included for debugging. I've also updated our contributors lists.

### CODE REVIEW

File: packages/3dom/package.json [hunk 1]
```diff
@@ -11,9 +11,12 @@
   "engines": {
     "node": ">=6.0.0"
   },
-  "author": "DevXR constributors",
-  "main": "./lib/context.js",
-  "module": "./lib/context.js",
+  "contributors": [
+    "Chris Joel <cdata@google.com>",
+    "Emmett Lalish <elalish@google.com"
```
Comment (sun765) [inline packages/3dom/package.json:16]: missing a ">" maybe?

File: packages/modelviewer.dev/package.json [hunk 2]
```diff
@@ -11,7 +11,14 @@
     "update:package-lock": "rm ./package-lock.json && npm i --only=production",
     "create-legacy-bundles": "node ./scripts/create-legacy-bundles.js"
   },
-  "author": "DevXR Contributors",
+  "contributors": [
+    "Jordan Santell <jsantell@google.com>",
+    "Chris Joel <cdata@google.com>",
+    "Emmett Lalish <elalish@google.com",
```
Comment (sun765) [inline packages/modelviewer.dev/package.json:17]: missing a ">" maybe?

File: packages/render-fidelity-tools/package.json [hunk 3]
```diff
@@ -13,7 +13,11 @@
     "update:package-lock": "rm ./package-lock.json && npm i --only=production",
     "clean": "rm -rf ./dist ./lib ./test/results ./test/renderers/filament/{cmgen,ktx}"
   },
-  "author": "DevXR Contributors",
+  "contributors": [
+    "Chris Joel <cdata@google.com>",
+    "Emmett Lalish <elalish@google.com",
```
Comment (sun765) [inline packages/render-fidelity-tools/package.json:18]: the same , ">"

File: packages/modelviewer.dev/package.json [hunk 4]
```diff
@@ -11,7 +11,14 @@
...
+    "Ricardo Cabello <ricardocabello@google.com>",
+    "Matt Small <mbsmall@google.com>",
+    "Yuin Chin <yuin@google.com>"
```
Comment (mrdoob) [inline packages/modelviewer.dev/package.json:20]: Yuin Chien

Comment (elalish) [reply packages/modelviewer.dev/package.json]: Thanks! That typo's been there a long time...

**Pull request merged**